src/app/forms/components/switchable-input/switchable-input.component.html

```diff
@@ -0,0 +1,86 @@
+<ng-container *ngIf="isEditing; else view" [ngSwitch]="type" [formGroup]="form">
+  <app-autocomplete
+    *ngSwitchCase="types.AUTOCOMPLETE"
+    [formControlName]="name"
+    [name]="name"
+    [label]="label"
+    [width]="width"
+    [options$]="options$"
+    [noBottomMargin]="true"
+  ></app-autocomplete>
+
+  <app-checkbox-group
+    *ngSwitchCase="types.CHECKBOX"
+    [formControlName]="name"
+    [name]="name"
+    [label]="label"
+    [width]="width"
+    [options]="options!"
```

Comment:
Seems this:
```js
@Input() options: Array<FormNodeOption<string | number | boolean>> = [];
```
in the component would be better than the `!` operator in html...

Reply:
After discussion it's probably ok as it is...